Documentation/ProjectFile/prj/processes/process/THERMO_HYDRO_MECHANICS/t_reference_temperature.md

```diff
@@ -0,0 +1 @@
+Reference temperature for the thermo-hydro-mechanical process.
```

Comment:
Reference for which part? Otherwise one might think it is not full __Thermo__ HydroMechanics...